src/components/footer/ContactWithIcon.tsx

```diff
@@ -0,0 +1,48 @@
+import {
+	LocationMarkerIcon,
+	MailIcon,
+	PhoneIcon,
+} from "@heroicons/react/outline";
+import clsx from "clsx";
+
+type ContactWithIconProps = {
+	iconType?: "Mail" | "MapPinPoint" | "Phone";
+	text: string;
+	className?: string;
+};
+
+const icons = {
+	Mail: <MailIcon />,
+	MapPinPoint: <LocationMarkerIcon className="text-sm" />,
+	Phone: <PhoneIcon />,
+};
+
+export function ContactWithIcon({
+	text,
+	className,
+	iconType,
+}: ContactWithIconProps) {
+	return (
+		<div className="flex flex-row justify-center items-center space-x-2">
+			<span className="w-6">{iconType && icons[iconType]}</span>
+			<span
+				className={clsx(
+					"font-bold text-center hover:text-pink-light",
+					className && className,
+				)}
+			>
+				{iconType === "Mail" && <a href={`mailto:${text}`}>{text}</a>}
+				{iconType === "MapPinPoint" && (
+					<a
+						target="_blank"
+						rel="noopener noreferrer"
+						href={`https://maps.google.com/?q=${text}`}
+					>
+						{text}
+					</a>
+				)}
+				{iconType === "Phone" && <span>{text}</span>}
```

Comment:
Ez is lehet link, `<a href={`tel:${text}`}>{text}</a>}`

Reply:
javítva.